README.md

```diff
@@ -4,128 +4,108 @@
 
 [Distributed TSDB](https://docs.m3db.io/m3db/) and [Query Engine](https://docs.m3db.io/how_to/query/), [Prometheus Sidecar](https://docs.m3db.io/integrations/prometheus/), [Metrics Aggregator](https://docs.m3db.io/overview/components/#m3-aggregator), and more such as [Graphite storage and query engine](https://docs.m3db.io/integrations/graphite/).
 
-More information:
+## Table of Contents:
+- [Install](#install)
+- [Dependencies](#dependencies)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Community meetings](#community-meetings)
+- [Office Hours](#office-hours)
```

Comment:
Try to be consistent with capitalizing headers

Reply:
@benraskin92 Good call, and something in the style guide anyway